.azure_pipelines/pytorch-tests-pipeline.yml

```diff
@@ -0,0 +1,37 @@
+# PyTorch PR PyTorch Tests Builds Pipeline on Azure DevOps
+#
+# This pipeline runs custom PyTorch unit-tests on PyTorch 
+# wheels generated during PR builds.
+
+stages:
+- stage: 'PRCustomTests'
+  displayName: 'Run custom unit tests on PyTorch wheels'
+  jobs:
+  - template: job_templates/build-verify-publish-template-unix.yml
+    parameters:
+      name: ubuntu_1804_CPU_docker
+      pool: 'PyTorch-Linux-CPU'
+      container_endpoint: pytorchms.azurecr.io
+      pytorch_tests_stage: True
+      customMatrixes:
+        Py_36:
```

Comment:
Idk if it matters but there's a mismatch between `Py_36` and `PYTHON_VERSION: 3.6` vs. configuration `py_38` and container_image `py_38`

Reply:
Yeah it doesn't matter actually, as the `pytorch_tests` conda env is what ends up being used, and its `PYTHON_VERSION:3.6` determines the actual Python version we use. I will actually rename our Docker images and remove the Python version, as it doesn't matter.

Reply:
In addition, we no longer use the `$configuration` env var, so I will remove these too.

Reply:
I plan to make CPU and CUDA 102 Docker images for the time being, as these are the Ubuntu configs we'll be testing on.